### PR TITLE
Update location-verification-API-Readiness-Checklist.md

### PR DESCRIPTION
**Pull request by jlurien**
#### What type of PR is this?

* documentation

#### What this PR does / why we need it:

User Story for location-verification is now available

### CODE REVIEW

**Review by bigludo7 — APPROVED** [review]
LGTM.

**Comment by hdamker** [comment]
The PR is correct, but is there a specific reason that the user story is in Supporting Documents and not in API Documentation as with other APIs? But guess that doesn't matter too much.

**Comment by jlurien** [comment]
> The PR is correct, but is there a specific reason that the user story is in Supporting Documents and not in API Documentation as with other APIs? But guess that doesn't matter too much.

I guess no particular reason. should it be moved to API Doc?

**Comment by jlurien** [comment]
User Story moved

**Review by maxl2287 — DISMISSED** [review]

**Review by bigludo7 — APPROVED** [review]
LGTM

**Comment by jlurien** [comment]
@hdamker for 

|  9 | Test result statement                        |   O   |         O         |    O    |    M   |tbd| link |

Do you know what is expected? Do we have any example from other APIs? Thanks

**Comment by bigludo7** [comment]
> @hdamker for
> 
> | 9 | Test result statement | O | O | O | M |tbd| link |
> 
> Do you know what is expected? Do we have any example from other APIs? Thanks

I had same issue in OTP validation API. I've pinged @trehman-gsma  for help - having a statement from GSMA that at least one implem has go thru the test could perhaps be an option?

**Comment by hdamker** [comment]
> Do you know what is expected? Do we have any example from other APIs? Thanks

I'm not aware of an example that any of the APIs has achieved this line within this release cycle. Suppose we accept this in the TSC for this cycle and have also to define this criteria more precise.

How it is [described](https://wiki.camaraproject.org/display/CAM/API+Release+Process#APIReleaseProcess-APIreadinesschecklist):

> A statement in a discussion issue of the API Sub Project by at least one of the API Sub Project members that the Gherkin feature files have been successfully executed against their (lab) API implementation. 

So not an implementation of a previous public version (which GSMA can have certified), but a (lab) implementation of the release candidate version from M3 with the test definition from the same version. The motivation was to reduce the risk that bugs in the API specification and inconsistencies between API and Test specification will be found only after the public release. 

But as the release candidates came late and the test definitions in many cases even later, there was no time to fulfil this criteria in the current cycle.

**Comment by jlurien** [comment]
I agree that is going to be hard to fully comply with this for this meta, due to the time constraints.

**Comment by jlurien** [comment]
@hdamker @bigludo7 how should we proceed with the PR regarding "Test result statement"? Do we open an issue in RM? Do we keep "tbd" in the meantime?

**Comment by hdamker** [comment]
> @hdamker @bigludo7 how should we proceed with the PR regarding "Test result statement"? Do we open an issue in RM? Do we keep "tbd" in the meantime?

Issue in RM to make them (and the TSC) aware makes sense for to me. The other APIs heading for stable have reported "N" as status, which is maybe more true, as there is no plan to get it done before the meta release.

**Comment by jlurien** [comment]
Created issue in RM: https://github.com/camaraproject/ReleaseManagement/issues/89

File: documentation/API_documentation/location-verification-API-Readiness-Checklist.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Checklist for location-verification v1.0.0-rc.1 in r1.1.
 |  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |Y| |
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |Y| |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |Y| inline in yaml |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |tbd| PR: /pull/227  |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |Y| /documentation/API_documentation/location-verification-User-Story.md  |
 |  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |Y| /code/Test_definitions/location-verification.feature |
 |  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |Y| /code/Test_definitions/location-verification.feature |
 |  9 | Test result statement                        |   O   |         O         |    O    |    M   |tbd| link |
```
Comment (jlurien) [inline documentation/API_documentation/location-verification-API-Readiness-Checklist.md:15]:
```suggestion
|  9 | Test result statement                        |   O   |         O         |    O    |    M   | N | camaraproject/ReleaseManagement/issues/89 |
```

File: documentation/API_documentation/location-verification-API-Readiness-Checklist.md [hunk 2]
```diff
@@ -9,14 +9,14 @@ Checklist for location-verification v1.0.0-rc.1 in r1.1.
...
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |Y| |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |Y| inline in yaml |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |tbd| PR: /pull/227  |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |Y| /documentation/API_documentation/location-verification-User-Story.md  |
 |  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |Y| /code/Test_definitions/location-verification.feature |
 |  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |Y| /code/Test_definitions/location-verification.feature |
-|  9 | Test result statement                        |   O   |         O         |    O    |    M   |tbd| link |
+|  9 | Test result statement                        |   O   |         O         |    O    |    M   | N | camaraproject/ReleaseManagement/issues/89 |
```
Comment (jlurien) [inline documentation/API_documentation/location-verification-API-Readiness-Checklist.md:15]:
```suggestion
|  9 | Test result statement                        |   O   |         O         |    O    |    M   |N| camaraproject/ReleaseManagement/issues/89 |
```

**Comment by jlurien** [comment]
Until a final resolution to the issue I have filled the row with "N" and a reference to the issue in the comments column

**Comment by jlurien** [comment]
@bigludo7 @maxl2287 please approve so I can merge

**Review by bigludo7 — APPROVED** [review]
LGTM

**Pull request merged**